common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -1,6 +1,7 @@
 package navigation
 
 import conf.Configuration
+import conf.switches.Switches
```

Comment:
Small thing: Instead of importing all the switches into this file, you could just import the one you are using `import conf.switches.Switches.SupportFrontendActive`